app/controllers/api/v1/printers_users_controller.rb

```diff
@@ -19,6 +19,11 @@ class PrintersUsersController < Api::V1::ApiController
 
       api :POST, '/v1/printers_users', "Create an printers_user"
       def create
+        @printers_user = PrintersUser
+          .where(user_id: params["printers_users"]["user_id"], tag: params["printers_users"]["tag"])
+          .first_or_initialize
+        @printers_user.printer_id = params["printers_users"]["printer_id"]
+
         save_and_render_object(@printers_user)
       end
```

Comment:
Unless there's a specific reason for doing it, I recommend not setting instance members if we can avoid it.

```suggestion
      api :POST, '/v1/printers_users', "Create an printers_user"
      def create
        user_id, printer_id, tag = params["printers_users"].values_at('user_id', 'printer_id', 'tag')
        save_and_render_object PrintersUser
          .where(user_id: user_id, tag: tag)
          .first_or_initialize do |printers_user|
             printers_user.printer_id = printer_id
             save_and_render_object(printers_user)
          end
      end
```